modules/monitoring-installation-progress.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-installations.adoc
+
+[id="monitoring-installation-progress_{context}"]
+= Monitoring installation progress
+
+You can monitor high level installation, bootstrap, and control plane logs as an {product-title} installation progresses. This provides greater visibility into how an installation progresses and helps identify the stage at which an installation failure occurs.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* You have SSH access to your hosts.
+* You have the fully qualified domain names of the bootstrap and master nodes.
++
+[Note]
```

Comment:
s/[Note]/[NOTE]/ should fix the formatting issue

Reply:
Resolved.